assets/src/admin/paired-browsing/app.css

```diff
@@ -0,0 +1,90 @@
+html,
+body {
+	margin: 0;
+	padding: 0;
+	height: 100%;
+	width: 100%;
+}
+
+body {
+	display: flex;
+	flex-direction: column;
+}
+
+body * {
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+}
+
+#header {
+	height: 32px;
+	background-color: #0075c2;
+}
+
+#header * {
+	color: #fff;
+}
+
+#header ul {
+	display: flex;
+	margin: 0 10px;
+	padding: 0 40px;
+	list-style: none;
+	align-items: center;
+	height: 100%;
+}
+
+#header li {
+	margin: 0 5px;
+}
+
+#header li:last-child {
+	margin-left: auto;
+}
+
+#exit-link {
+	text-decoration: none;
```

Comment:
The exit link currently does not show any sign of being clickable, even when hovering over it.
Maybe add the underline back on hover?

Reply:
I thought the cursor becoming a pointer was enough, but yes an underline on hover would make it more understandable.